content/javascript/concepts/arrays/terms/includes/includes.md

```diff
@@ -0,0 +1,73 @@
+---
+Title: '.includes()'
+Description: 'Returns true if a given value is included in an array'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `.includes()` method returns `true` if a given value is included in an array. Otherwise, it returns `false`.
+
+## Syntax
+
+```pseudo
+array.includes(value, index=0);
+```
+
+The following parameters are used:
+
+- A case-sensitive and type-sensitive `value` that is checked for inclusion in the `array`.
+- An optional `index`, defaulted to 0, that tells `.includes()` where to begin the check.
+
+## Example
+
+The `.includes()` method can be used in a few ways. First, it can be used directly with an array:
+
+```js
+[1, 2, 3].includes(3);
+// Output: true
+```
+
+It can also be used with a variable assinged with an array:
+
+```js
+const myArray = [1, '2', 3];
+
+myArray.includes(2);
+// Output: falsee
+```
+
+The reason the output above was `false` is because `.includes()` is type-sensitive and it checked for a number-type value 2 whereas `myArray` has no such value. Instead, it has a string literal that reads as "2".
+
+Some edge-cases to keep in mind about `.includes()`:
```

Comment:
Do you feel the edge cases are best written here? I'm wondering if it would be better at the end of the "syntax" section as its own level 3 subheading. 🤔